From 0747494a45986d14f32dc6efdde84fd5b5c04465 Mon Sep 17 00:00:00 2001
Message-ID: <0747494a45986d14f32dc6efdde84fd5b5c04465.1699318202.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1699318202.git.sharlatanus@gmail.com>
References: <cover.1699318202.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 6 Nov 2023 22:18:29 +0000
Subject: [PATCH 12/16] gnu: cl-containers: Update to 0.12.1-4.781ebfe.

* gnu/packages/lisp-xyz.scm (sbcl-cl-containers): Update to 0.12.1-4.781ebfe.
[source]: Update URL as maintainer and owner of repository was changed.

Change-Id: I7db432378aeb03cd3ac0336efc5607522aced06c
---
 gnu/packages/lisp-xyz.scm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 1cec0661a5..e001ccd703 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -9248,8 +9248,8 @@ (define-public ecl-metatilities-base
   (sbcl-package->ecl-package sbcl-metatilities-base))
 
 (define-public sbcl-cl-containers
-  (let ((commit "3d1df53c22403121bffb5d553cf7acb1503850e7")
-        (revision "3"))
+  (let ((commit "781ebfe0888bae46f07c018f7d473898b1bd4f5f")
+        (revision "4"))
     (package
       (name "sbcl-cl-containers")
       (version (git-version "0.12.1" revision commit))
@@ -9257,12 +9257,12 @@ (define-public sbcl-cl-containers
        (origin
          (method git-fetch)
          (uri (git-reference
-               (url "https://github.com/gwkkwg/cl-containers")
+               (url "https://github.com/hraban/cl-containers")
                (commit commit)))
          (file-name (git-file-name "cl-containers" version))
          (sha256
           (base32
-           "18s6jfq11n8nv9k4biz32pm1s7y9zl054ry1gmdbcf39nisy377y"))))
+           "1nrql8s1j123v5gscy99lxvhlzp0ijig9x94w30v3lwfa58hf90l"))))
       (build-system asdf-build-system/sbcl)
       (native-inputs
        (list sbcl-lift))
-- 
2.41.0

